stories/FileMetadata.stories.tsx

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable @typescript-eslint/camelcase */
```

Comment:
Why is this needed now? 🤔 (I see eslint complaining too)

Reply:
There is some ghost in the machine but I haven't tried to track it down. My hunch is that the eslint rule gets applied in a later pass and there's a bug that relies on the order of loading files (or something like that).